src/loaders/JsonImageInfoLoader.ts

```diff
@@ -90,24 +90,31 @@ const convertImageInfo = (json: JsonImageInfo): ImageInfo => ({
 });
 
 class JsonImageInfoLoader implements IVolumeLoader {
-  imageInfo: JsonImageInfo | null = null;
-  imageArray: PackedChannelsImage[] = [];
+  urls: string[];
+  time: number;
+  jsonInfo: JsonImageInfo | null = null;
+
+  constructor(urls: string | string[]) {
+    if (Array.isArray(urls)) {
+      this.urls = urls;
+    } else {
+      this.urls = [urls];
+    }
 
-  async getImageInfo(loadSpec: LoadSpec): Promise<JsonImageInfo> {
-    if (!this.imageInfo) {
-      const response = await fetch(loadSpec.url);
-      const imageInfo = (await response.json()) as JsonImageInfo;
+    this.time = 0;
+  }
 
-      imageInfo.pixel_size_unit = imageInfo.pixel_size_unit || "μm";
-      this.imageInfo = imageInfo;
+  async getJsonImageInfo(loadSpec: LoadSpec): Promise<JsonImageInfo> {
+    const response = await fetch(this.urls[loadSpec.time]);
```

Comment:
I like this.  It also makes me wonder if we should support something similar for other file formats when the T samples have been split among many files.

Reply:
I mean basically the only reason I didn't do this for `TiffLoader` as well is that none of the example datasets required it.